src/app/[locale]/(profile)/_components/change-password-form.tsx

```
@@ -0,0 +1,140 @@
+import { changePassword } from "../_actions/change-password.actions";
+import { useTranslations } from "next-intl";
+import { FormControl, FormField, FormItem, FormLabel, FormMessage } from "@components/ui/form";
+import { Input } from "@components/ui/input";
+import { toast } from "sonner";
+
+export default function ChangePasswordForm() {
+  // Translation
+  const t = useTranslations("change-password");
+
+  // Hooks
+  const { mutate, isPending } = useChangePassword();
```

ESLint

[error] 'mutate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)